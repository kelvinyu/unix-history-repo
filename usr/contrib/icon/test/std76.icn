record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("1 & 2 & 3 & 4 ----> ",image(1 & 2 & 3 & 4) | "none")
   write("(1 & 2 & 3 & x) := 3 ----> ",image((1 & 2 & 3 & x) := 3) | "none")
   write("x ----> ",image(x) | "none")
   write("(x & 2 & 3 & 4) := 3 ----> ",image((x & 2 & 3 & 4) := 3) | "none")
end
