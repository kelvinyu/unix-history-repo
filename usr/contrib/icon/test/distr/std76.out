1 & 2 & 3 & 4 ----> 4
(1 & 2 & 3 & x) := 3 ----> 3
x ----> 3
Run-time error 111 at line 12 in std76.icn
variable expected
offending value: 4
